src/components/Time/TimeControls.tsx

```diff
@@ -5,14 +5,14 @@ import RefreshContainer from 'components/Refresh/Refresh';
 
 type ReduxProps = {};
 
-type TimeRangeProps = ReduxProps & {
+type TimeControlsProps = ReduxProps & {
   disabled: boolean;
   id: string;
 
   handleRefresh: () => void;
 };
 
-export class TimeRange extends React.PureComponent<TimeRangeProps> {
+export class TimeControls extends React.PureComponent<TimeControlsProps> {
```

Comment:
@jshaughn what do you think about this renaming of `TimeRange` component to `TimeControls`? Or do you have other suggestions?
There's two motivations:
- first because it's not just a range, it also manages the refresh rate. And your PR will also add the replay option. So much more than just a range
- this is also because I wanted a name for my new from/to pickers component, and TimeRange looks actually not too bad for that, I think.
Or any other suggestion?

Reply:
@jotak I agree that that is a better name given the evolving nature of the component.  We can rename to "TimeControls" and free up "TimeRange" for another purpose.   Should I do it in my PR or will you make it part of your work?

Reply:
As you want. I'm already doing that here, so you can leave it untouched

Reply:
I will defer the change, thanks.